docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -325,6 +325,7 @@ Create a container
              "CapDrop": ["MKNOD"],
              "GroupAdd": ["newgroup"],
              "RestartPolicy": { "Name": "", "MaximumRetryCount": 0 },
+             "AutoRemove": true,
```

Comment:
Can you also add a description in the `JSON parameters` section below? https://github.com/docker/docker/pull/20848/files#diff-18fd06f3efcea92bb87641d8363b1d08R462, something like:

```Markdown
-   **AutoRemove** - Boolean value, set to `true` to automatically remove the container
     when the container's process exits, or the client disconnects.
``` 

(please check if the description is correct)